cpplib/Tests/MainTest.cpp

```
@@ -93,6 +97,182 @@ struct FMIC_TS {
+//	// 2. Use cutoff for prepare PointsSoA
+//	cpplib::PointsSoA<double> psoa;
+//	psoa.initializeWithPeriodicImages(buildresult.atoms.points, buildresult.atoms.types, cell.fracToCart(), cell.cartToFrac(), ED_cutoff);
+//
+//
+//	std::ofstream of("out.txt");
+//	for (size_t i = 0; i < 100; i++)
+//	{
+//		for (size_t j = 0; j < 100; j++)
+//		{
+//			for (size_t k = 0; k < 100; k++)
+//			{
```

Cppcheck (2.21.0)

[performance] Function parameter 'plane' should be passed by const reference.

(passedByValue)